mk/build.sh

```diff
@@ -72,14 +73,57 @@ XENADMIN_DIR="$( cd -P "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
 
 source ${XENADMIN_DIR}/mk/declarations.sh
 
+if test -z "${XC_BRANDING}"; then XC_BRANDING=citrix; fi
```

Comment:
Oops. I badly misread that merge conflict - I'll revert that commit and do a more careful merge.